versionedRouter.js

```diff
@@ -457,7 +457,7 @@ async function routerHandleDest(ctx) {
         if (routerDestHandler.processMetadataForRouter) {
           listOutput.forEach(output => {
             output.metadata = routerDestHandler.processMetadataForRouter(
-              output
```

Comment:
Sometimes, we might need data from objects other than `metadata` in the `output` object. Let's keep it generic, and the destination handler take care of unwrapping it.

Reply:
Updated